[2022-12-04T16:12:42.942+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.notification scheduled__2022-12-03T09:00:00+00:00 [queued]>
[2022-12-04T16:12:42.949+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.notification scheduled__2022-12-03T09:00:00+00:00 [queued]>
[2022-12-04T16:12:42.950+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:12:42.952+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-04T16:12:42.953+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:12:42.967+0000] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): notification> on 2022-12-03 09:00:00+00:00
[2022-12-04T16:12:42.976+0000] {standard_task_runner.py:55} INFO - Started process 278 to run task
[2022-12-04T16:12:42.980+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_generation', 'notification', 'scheduled__2022-12-03T09:00:00+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/generate_data.py', '--cfg-path', '/tmp/tmp1hva8ppy']
[2022-12-04T16:12:42.984+0000] {standard_task_runner.py:83} INFO - Job 22: Subtask notification
[2022-12-04T16:12:43.055+0000] {task_command.py:389} INFO - Running <TaskInstance: data_generation.notification scheduled__2022-12-03T09:00:00+00:00 [running]> on host c7dedbf79ea9
[2022-12-04T16:12:43.129+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_generation
AIRFLOW_CTX_TASK_ID=notification
AIRFLOW_CTX_EXECUTION_DATE=2022-12-03T09:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-03T09:00:00+00:00
[2022-12-04T16:12:43.132+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-12-04T16:12:43.134+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo received data at 2022-12-03']
[2022-12-04T16:12:43.149+0000] {subprocess.py:86} INFO - Output:
[2022-12-04T16:12:43.153+0000] {subprocess.py:93} INFO - received data at 2022-12-03
[2022-12-04T16:12:43.154+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2022-12-04T16:12:43.184+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=data_generation, task_id=notification, execution_date=20221203T090000, start_date=20221204T161242, end_date=20221204T161243
[2022-12-04T16:12:43.233+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-04T16:12:43.252+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
